lib/quby/questionnaires/dsl/table_builder.rb

```diff
@@ -39,6 +40,18 @@ def question(key, options = {}, &block)
           @table.items << question
           @panel.items << question
         end
+
+        def method_missing(method_sym, *args, &block)
```

Comment:
onnodige duplicatie

Je kunt een module maken die in initialize(*args) extract_options! aanroept om de custom_methods te krijgen en deze 2 methods definieert. Moet je in de initializers super aanroepen.